public/index.html

```diff
@@ -30,7 +30,7 @@
             const botConnection = new BotChat.DirectLine({
                 secret: params['s'],
                 token: params['t']
-                }, params["domain"]
+                }, params["domain"] //the domain should be the DL url https://directline.botframework.com/v3/directline
```

Comment:
This is not correct. The purpose of this optional parameter is to override the default domain you specified.